tasks/README.md

```diff
@@ -104,8 +107,13 @@ If a parameter type accepts `null` the task runner will accept either a `null` v
 
 ### Metaparameters
 
-In addition to the tasks parameters, the task runner may inject metaparameters prefixed by '_'. These include `_noop` and `_task`.
+In addition to the tasks parameters, the task runner may inject metaparameters prefixed by `_`.
+
+**_noop**: Used to implement logic in tasks to support cases where task should not perform certain actions.
 
+**_task**: Allow multiple task implementations to access the same executable file. The `_task` metaparameter provides the executable the task name to allow task specific logic to be implemented. 
```

Comment:
Personally I'd like this to exclude the module name, i.e. `package::status` would define `_task=status`, `package` would define `_task=init`. We weren't particularly clear before, and having to include the module name in the comparison (when it will always be the same module name) is annoying. This would imply a change to Bolt's existing behavior.

Reply:
+ 1 for this. Anybody think of a reason not to do it this way?